### PR TITLE
ENH: component.doc

### LINKED ISSUE #167: Docstring as Tooltip in TyphonSignalPanel
Use the component docstring as the tooltip in `TyphonSignalPanel`

### PR DESCRIPTION
**Pull request by klauer**
Suggested as a follow-up to #174 by @teddyrendahl 

Use Component docstrings when creating Typhon widgets

Closes #167 

Example screenshot:
image

### CODE REVIEW

File: typhon/signal.py [hunk 1]
```diff
@@ -293,19 +293,16 @@ def _set_layout(self):
         shown_kind = [kind for kind in Kind if self._kinds[kind.name]]
         # Iterate through kinds
         signals = list()
-        for kind in Kind:
-            if kind in shown_kind:
```
Comment (teddyrendahl) [inline typhon/signal.py]: Why did I think I needed this? Geez.

File: typhon/utils.py [hunk 2]
```diff
@@ -53,7 +56,7 @@ def is_signal_ro(signal):
 
 
 def grab_kind(device, kind):
-    """Grab all signals of a specific Kind from a Device"""
+    """Grab all signals of a specific Kind from a Device instance"""
```
Comment (teddyrendahl) [inline typhon/utils.py]: Mind just putting a docstring specifying the returned value? Should have been there before but I figured if I documented them poorly I'd struggle less breaking the API 😄 This seems generally useful outside `typhon`. I like having the `Component` available as well.

File: typhon/signal.py [hunk 3]
```diff
@@ -147,7 +152,7 @@ def add_signal(self, signal, name):
         """
         logger.debug("Adding signal %s", name)
         # Create the read-only signal
-        read = signal_widget(signal, read_only=True)
+        read = signal_widget(signal, read_only=True, tooltip=tooltip)
```
Comment (teddyrendahl) [inline typhon/signal.py]: So this goes on the read only widget? I kind of imagined it would go on the `QLabel`. That is created in `_add_row`

Comment (klauer) [reply typhon/signal.py]: Good call. 👍

**Comment by klauer** [comment]
Rebased and force-pushed to remove `fake_epics_pv`; screenshot in PR text.

**Comment by ZLLentz** [comment]
I like this a lot

**Pull request merged**